### ISSUE TITLE
Multi knob clicky in 24dB mode

### ISSUE THREAD

**Issue by mkruselj** [reales/OB-Xd#18]
Not sure if this is a known issue or not, but this seems to have been forever in OB-Xd.

Easiest way to test: set Cutoff to 9 o'clock, resonance at 12 o'clock, no envelope, and enable 24 dB mode. Hold a note then tweak the Multi knob fast (or host automate it, etc.). You can hear clicks which shouldn't happen ideally.

Is there a way to smooth this out so that it doesn't click? Thanks!

**Comment by reales** [comment]
Right, looks like Multi parameter needs some interpolation to sound without clicks on value change.

**Comment by mkruselj** [comment]
I hope that is a piece of cake to do. It will be great once it's done. Thanks!
